### PR TITLE
Add disclaimer about RetentionComplianceAction of Keep

### PR DESCRIPTION
**Pull request by TreYager**
User Story 1953376: Update Public Documentation for SCC Retention Tag flow, RetentionComplianceAction Keep does not become a Tag

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-retention-policies.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ By choosing the settings for retaining and deleting content in your retention po
 
 - Retain-only
 
-    For this configuration, choose **Retain items for a specific period** and **At end of the retention period: Do nothing**. Or, select **Retain items forever**.
+    For this configuration, choose **Retain items for a specific period** and **At end of the retention period: Do nothing**. Or, select **Retain items forever**. Note that this action type is treated as a hold, so Retention Policy tags are not displayed when the message is viewed in Outlook.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-retention-policies.md:242]:
```suggestion
    For this configuration, choose **Retain items for a specific period** and **At end of the retention period: Do nothing**. Or, select **Retain items forever**. Note that this action type is treated as a hold, so retention policy tags are not displayed when the message is viewed in Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @TreYager, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@TreYager Gentle reminder, please implement the suggested changes. Thanks

**Comment by TreYager** [comment]
Sorry guys, this is the first doc update I've done here. Do you need me to do anything else?

**Comment by cabailey** [comment]
@TreYager  I believe you're internal to Microsoft?  These docs are actively maintained so we can update them more efficiently if you contact me direct (Teams or email).

**Comment by cabailey** [comment]
Closing - taking this offline for now

**Pull request closed without merging**